curriculum/challenges/english/03-front-end-libraries/redux/use-const-for-action-types.english.md

```diff
@@ -35,8 +35,8 @@ tests:
   - text: The <code>authReducer</code> function should handle multiple action types with a switch statement.
     testString: getUserInput => assert((function() { return typeof authReducer === 'function' && getUserInput('index').toString().includes('switch') && getUserInput('index').toString().includes('case') && getUserInput('index').toString().includes('default') })());
   - text: <code>LOGIN</code> and <code>LOGOUT</code> should be declared as <code>const</code> values and should be assigned strings of <code>LOGIN</code>and <code>LOGOUT</code>.
-    testString: getUserInput => assert((function() {  const noWhiteSpace = getUserInput('index').toString().replace(/\s/g,''); return (noWhiteSpace.includes('constLOGIN=\'LOGIN\'') || noWhiteSpace.includes('constLOGIN="LOGIN"')) && (noWhiteSpace.includes('constLOGOUT=\'LOGOUT\'') || noWhiteSpace.includes('constLOGOUT="LOGOUT"')) })());
-  - text: The action creators and the reducer should reference the <code>LOGIN</code> and <code>LOGOUT</code> constants.
+    testString: getUserInput => assert((function() { const noWhiteSpace = code.replace(/\s/g, ''); assert(/constLOGIN=('"`)LOGIN\1/.test(nonWhiteSpace) && /constLOGOUT=('"`)LOGOUT\1/.test(nonWhiteSpace)) }());
```

Comment:
@AshNaz87 Part of the reason your PR is failing the Travis build is you did not copy/paste what I posted in the forum discussion.  The other part, is that I put `nonWhiteSpace` instead of `noWhiteSpace` for the variable name.  Also, my regex was slightly off (I had not tested the code I posted).  The capture group values **'"`** should have been inside a character set using `[` and `]`.  I have corrected all of these issues in the suggestion below.  Please commit this change (that I have tested locally), so the build will pass,
```suggestion
    testString: const noWhiteSpace = code.replace(/\s/g, ''); assert(/constLOGIN=(['"`])LOGIN\1/.test(noWhiteSpace) && /constLOGOUT=(['"`])LOGOUT\1/.test(noWhiteSpace));
```